docs/reference/testing/testing-framework.asciidoc

```diff
@@ -42,25 +36,18 @@ First, you need to include the testing dependency in your project. If you use ma
     <scope>test</scope>
     <type>test-jar</type>
   </dependency>
-  <dependency>
-    <groupId>org.elasticsearch</groupId>
-    <artifactId>elasticsearch</artifactId>
-    <version>${elasticsearch.version}</version>
-    <scope>test</scope>
-  </dependency>
 </dependencies>
 --------------------------------------------------
 
-Replace the elasticsearch version and the lucene versions with the current elasticsearch version and its accompanying lucene release.
-And replace the "randomizedtesting version" with the version that the current elasticsearch uses.
+Replace the elasticsearch version and the lucene versions with the corresponding elasticsearch version and its accompanying lucene release.
 
-There are already have a couple of classes, you can inherit from in your own test classes. The advantages of doing so is having already defined loggers, the whole randomized infrastructure is set up already.
+There are already have a couple of classes, you can inherit from in your own test classes. The advantages of doing so is having already defined loggers as well as the whole randomized infrastructure is set up already together with a lot of helper methods.
```

Comment:
We provide a few classes that you can inherit from in your own test classes which provide:
* pre-defined loggers
* randomized testing infrastructure
* a number of helper methods